### PR TITLE
Added 'default' option to migrations page + added code samples to Pha…

### PR DESCRIPTION
**Pull request by LiamMartens**
Added "default" column option to migrations documentation page and added code samples to Phalcon_Db_Index api reference

### CODE REVIEW

**Comment by LiamMartens** [comment]
@sergeyklay 
Sorry my bad, I removed the changes from the docs fork and created a pull request on 3.1.x for the docblock
https://github.com/phalcon/cphalcon/pull/12528

File: en/api/Phalcon_Db_Index.rst [hunk 1]
```diff
@@ -10,7 +10,6 @@ Class **Phalcon\\Db\\Index**
 
 Allows to define indexes to be used on tables. Indexes are a common way to enhance database performance. An index allows the database server to find and retrieve specific rows much faster than it could do without an index
 
-
```
Comment (sergeyklay) [inline en/api/Phalcon_Db_Index.rst]: Also please return back this

**Comment by LiamMartens** [comment]
@sergeyklay restored Db_Index to original state and added "default" line to all other languages

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**